cognite/pygen/cli.py

```diff
@@ -45,7 +46,16 @@ def generate(
             ),
             version: str = typer.Option(default=loaded_settings.version.default, help=loaded_settings.version.help),
             tenant_id: str = typer.Option(
-                default=loaded_settings.tenant_id.default, help=loaded_settings.tenant_id.help
+                default=loaded_settings.tenant_id.default, help=loaded_settings.tenant_id.help,
+            ),
+            token_url: str = typer.Option(
+                default=loaded_settings.token_url.default, help=loaded_settings.token_url.help
+            ),
+            scopes: str = typer.Option(
+                default=loaded_settings.scopes.default, help=loaded_settings.scopes.help
+            ),
```

Comment:
Do we need this one? 
Can't it just be set as `[f"https://{cdf_cluster}.cognitedata.com/.default"],`?

Reply:
Yes, it can be anything when not using Azure AD